components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadNotification.kt

```diff
@@ -86,12 +92,15 @@ internal object DownloadNotification {
     /**
      * Build the notification to be displayed when a download finishes.
      */
-    fun createDownloadCompletedNotification(context: Context, downloadState: DownloadState): Notification {
+    fun createDownloadCompletedNotification(context: Context, downloadJobState: DownloadJobState): Notification {
         val channelId = ensureChannelExists(context)
+        val downloadState = downloadJobState.state
 
         return NotificationCompat.Builder(context, channelId)
             .setSmallIcon(R.drawable.mozac_feature_download_ic_download_complete)
             .setContentTitle(downloadState.fileName)
+            .setWhen(downloadJobState.createdTime)
```

Comment:
I am wondering if this should be a separate timestamp i.e. the time the notification happened ( == the time the download was completed, not created)?